### PR TITLE
Harness reworking

### PR DESCRIPTION
**Pull request by sungam3r**
- remove not working GraphiQL, add Playground
- fix HTTP status code to always return 200 OK
- use IOptions pattern and values from config in middleware

### CODE REVIEW

File: src/GraphQL.Harness/GraphQLSettings.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace Example
 {
     public class GraphQLSettings
     {
-        public PathString Path { get; set; } = "/api/graphql";
+        public PathString GraphQLPath { get; set; }
```
Comment (sungam3r) [inline src/GraphQL.Harness/GraphQLSettings.cs]: Renamed as conflicted with embedded PATH property.

**Comment by joemcbride** [comment]
The GraphiQL _does_ work, last I checked, you just need to run the webpack to build the JavaScript bundle. So if we’re removing tat GraphiQL setup we should also remove that webpack config file.  I don’t necessarily mind using the Playground (it is nicer).

**Comment by sungam3r** [comment]
I did not know about webpack. Do you mean webpack.config.js ? Any other files to delete?
I agree that Playground is nicer. In addition, it requires less config/code files to work which may be useful as an example.

**Comment by sungam3r** [comment]
I removed webpack config file and also added GraphiQL support.

**Pull request merged**